tools/dump_kubernetes.sh

```diff
@@ -46,6 +46,10 @@ usage() {
   error '  -m, --max-bytes          max total bytes, 0=no limit, default='${DEFAULT_MAX_LOG_BYTES}
   error '  -l, --label              if set, dump logs only for pods with given labels e.g. "-l app=pilot -l istio=galley"'
   error '  -n, --namespace          if set, dump logs only for pods in the given namespaces e.g. "-n default -n istio-system"'
+  error '  -p, --pod                if set, dump logs only for pod matching these values e.g. "-n istio-system -p istio-policy-7857db5f54-wj59d -p istio-pilot-5cb9d848b5-xx25q"'
+  error '  -c, --control-plane      if set, dump debugging information from control plane'
+  error '  -u, --cluster            if set, dump logs from cluster'
+  error '  -x, --proxy              if set, dump logs from proxis e.g "-x istio=ingressgateway"'
```

Comment:
```
$ ./tools/dump_kubernetes.sh -x app=shell
Fetching http://localhost:15000/config_dump from
error: pod or type/name must be specified
Fetching http://localhost:15000/clusters from
error: pod or type/name must be specified
Fetching http://localhost:15000/stats from
error: pod or type/name must be specified
Fetching http://localhost:15000/certs from
error: pod or type/name must be specified
Fetching http://localhost:15000/memory from
error: pod or type/name must be specified
error: You must provide one or more resources by argument or filename.
Example resource specifications include:
   '-f rsrc.yaml'
   '--filename=rsrc.json'
   '<resource> <name>'
   '<resource>'
Wrote to istio-dump
```

Reply:
Yes, it's a bug, it did not manage to find app=shell pods in istio namespace. Problem is that  istio-systems is hardcoded for this dump for now, I'll fix that, but then this -x command  might require a namespace (-n istio-systems for the ingressgw for example?)